### PR TITLE
Translations of privacy strings in pegasus, apps, and dashboard

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

**Comment by tanyaparker** [comment]
@Hamms I hope this doesn't conflict with your syncs. Also, do you have a recommendation for how to add these translations to crowdin before your next sync (so they don't get overwritten?) I was going to upload all the files manually via the Crowdin UI but wondering if you have an easier way with your scripts.

**Comment by Hamms** [comment]
Hmm, it might. Lemme merge the latest sync, and if GH doesn't have a conflict then I think we're good!

**Comment by tanyaparker** [comment]
No problem. I think it'll be easy to fix anyway. I'm on it!

**Comment by Hamms** [comment]
Looks like it does; how difficult will it be to re-apply these changes to staging latest?

Re; uploading translations, the crowdin CLI has an `upload translations` command that should work the same as uploading via UI. You can apply it like:

    crowdin --config bin/i18n/codeorg_crowdin.yml --identity bin/i18n/codeorg_credentials.yml upload translations

provided you've set up credentials [as outlined here](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/i18n/codeorg_crowdin.yml#L7-L9)

Although I'm not sure what that will do for new translations we've received since the last sync; I suspect doing an upload either with the command or manually has the chance to loose use some work unless we do it right after a sync

**Comment by tanyaparker** [comment]
@Hamms Ok, I'll try that! Yes, I think uploading the professional translations will "overwrite" over volunteer translations. I think the volunteer translations would still exist, but the professional translations would take precedence because they're newer. But while I was doing this there wasn't much already translated, since the strings have only been in Crowdin a few days. I think it's ok. I'll be looking closely at your next sync to make sure we didn't lose any professional translations.

Merge conflict resolved. Ready for your thumbs up, Elijah. :)

File: apps/i18n/common/ko_kr.json [hunk 1]
```diff
@@ -534,11 +534,11 @@
...
-  "gdprDialogHeader": "Do you agree to using a website based in the United States?",
-  "gdprDialogDetails": "Data from your use of this site may be sent to and stored or processed in the United States.",
-  "gdprDialogSeePrivacyPolicy": "For details see our privacy policy",
-  "gdprDialogLogout": "Log out",
-  "gdprDialogYes": "예",
+  "gdprDialogHeader": "미국에 기반을 두고 있는 웹사이트 사용에 동의하십니까?",
+  "gdprDialogDetails": "귀하의 본 사이트 사용에 따른 데이터는 미국으로 전송되어 보관 또는 처리될 수 있습니다.",
+  "gdprDialogSeePrivacyPolicy": "(상세한 내용은 당사의 개인 정보 보호 정책을 참조하십시오.)",
+  "gdprDialogLogout": "로그 아웃",
+  "gdprDialogYes": "네",
```
Comment (tanyaparker) [inline apps/i18n/common/ko_kr.json]: Here's one that was already translated. According to google translate it looks like a Yes/Yeah. I'll revert and take the existing volunteer translation.

File: pegasus/cache/i18n/ru-RU.yml [hunk 2]
```diff
@@ -483,9 +483,10 @@
   homepage_header_donors: Code.org благодарит своих помощников
   homepage_donors_seall: Просмотреть всё
   homepage_donors_donate: Пожертвовать
-  homepage_below_hero_announcement_text: We care about your privacy and are making
-    some updates to our privacy policy, effective May 25.
-  homepage_below_hero_announcement_link_text: Узнать больше
+  homepage_below_hero_announcement_text: Мы соблюдаем вашу конфиденциальность и
+    вносим некоторые обновления в нашу политику конфиденциальности, которые вступают
+    в силу с 25 мая.
+  homepage_below_hero_announcement_link_text: Подробнее
```
Comment (tanyaparker) [inline pegasus/cache/i18n/ru-RU.yml:489]: Second instance where the string was already translated. Professional translations is briefer (better for buttons) and according to Google Translate says "Learn More" while the volunteer translation is more like "to learn more." Will take the professional translation.

File: pegasus/cache/i18n/zh-TW.yml [hunk 3]
```diff
@@ -391,9 +391,9 @@
   homepage_header_donors: Code.org 感謝所有支持者
   homepage_donors_seall: 檢視全部
   homepage_donors_donate: 贊助
-  homepage_below_hero_announcement_text: We care about your privacy and are making
-    some updates to our privacy policy, effective May 25.
-  homepage_below_hero_announcement_link_text: 繼續學習
+  homepage_below_hero_announcement_text: 我們尊重您的隱私權，並且正在對我們的隱私權政策做一些更新
+    更新後的政策將於 5 月 25 日生效。
+  homepage_below_hero_announcement_link_text: 瞭解更多資訊
```
Comment (tanyaparker) [inline pegasus/cache/i18n/zh-TW.yml:396]: Third instance. Volunteer translation is more like "continue studying" while professional translation is "learn more." Keeping professional translation.

**Comment by tanyaparker** [comment]
There were 3 strings that already had a volunteer translation. For the Korean string, I kept the volunteer translation that said "Yes" instead of the professional translation "Yeah." The other two I kept the professional translation which seemed more accurate/terse.

**Comment by Hamms** [comment]
Oh, to clarify I'm actually worried about all the _other_ strings. I believe the upload is an all-or-nothing thing, so any strings anywhere else in our system that have been updated will be overwritten with whatever value is currently in the i18n files.

**Pull request merged**